hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenTransactSpecs.java

```diff
@@ -1854,6 +1874,9 @@ public HapiSpec nestedHtsCaseStudy() {
                         getAccountBalance(DEFAULT_PAYER).hasTokenBalance(feeToken, Long.MAX_VALUE - 1_000L));
     }
 
+    // fails with
+    // Wrong status! Expected INSUFFICIENT_SENDER_ACCOUNT_BALANCE_FOR_CUSTOM_FEE, was INVALID_TOKEN_ID!
+    // @HapiTest
```

Comment:
The failure reason is good to write down, but I'm not sure we want it in the code itself, especially since this could fail differently depending on what other bugs are fixed

Reply:
I removed the comments